sample/client_credentials_sample.py

```diff
@@ -1,8 +1,8 @@
-﻿import json
+import adal
+import json
 import logging
 import os
 import sys
-import adal
```

Comment:
Historically, this code base only loosely adopts PEP8 and using pylint. And [PEP8 only enforces grouping, without requiring the order within the groups](https://softwareengineering.stackexchange.com/questions/341001/python-import-order).

So, if you are really going to adjust this, I would suggest you to just add a blank line in front of `import adal`, and then we will happily merge it. Alternatively, you could choose to abandon this PR.

Reply:
Thanks for the additional feedback here and insight into style for this code base. I will close this one.